OpenEdXMobile/src/main/java/org/edx/mobile/module/prefs/UserPrefs.java

```diff
@@ -46,18 +50,35 @@ public boolean isDownloadOverWifiOnly() {
         return onlyWifi;
     }
 
+    public boolean isSDCardDownloadEnabled(){
+        if (!environment.getConfig().isSDCardDownloadEnabled()){
```

Comment:
This check shouldn't be in this function. This function should only tell about the user prefs status.